Clarify the game rules

dmytroshytikov opened the pull request:
This file is a much better place for all this information.

dmytroshytikov committed:
Clarify the game rules

This file is a much better place for all this information.

Mikolaj commented:
Thank you! Is the an alternative to https://github.com/AllureOfTheStars/Allure/pull/122 or an update version of the old PR? In the latter case, I guess it would have been easier for you to force-push (`push -f`) an amended commit (`git commit -amend -a`) to the old PR instead. But no harm done, of course.

Mikolaj commented:
BTW, have a look at the CI -- it yelled at you. :D

dmytroshytikov commented:
> BTW, have a look at the CI -- it yelled at you. :D

So far, can't see. I will see if I can observe the issue again.

Mikolaj commented:
I mean, the "Some checks were not successful" reports problems. I think they are about exceeding 80 characters.

Mikolaj commented:
The "Some checks were not successful" is CI.

Mikolaj commented:
"Continuous integration" -- compiling, checking and testing after each push.

dmytroshytikov commented:
> I mean, the "Some checks were not successful" reports problems. I think they are about exceeding 80 characters.

Yeah, I see.
Well, I thought that as it is a text file (the markdown type), I can just type in the text I wanted, and the machine will automatically resize it as it should be. What should I do with it?
And when I went to the file, I can see changes I suggested there.

Mikolaj commented:
Markdown is a double purpose format. Some people view it formatted: 

https://github.com/AllureOfTheStars/Allure/blob/b0c0ffad460100aadf4d5ba3271ec4f9b1c1f68f/GameDefinition/PLAYING.md

other view it raw, e.g, in the console with a `cat` or `less` command

https://raw.githubusercontent.com/AllureOfTheStars/Allure/b0c0ffad460100aadf4d5ba3271ec4f9b1c1f68f/GameDefinition/PLAYING.md

Additionally, the latter form is used in the game itself. The help screen ends with this file included (as captured at compilation time), in raw form, and the start menu displays the initial portion, slightly formatted.

I'm afraid, to keep the 80 columns, one needs to use an old fashioned text editor with monospace font and a ruler or window size set to 80 columns. Then commit the changes using a commandline git. Alternatively, perhaps Visual Studio can do that, but I'm not familiar with it.

dmytroshytikov commented:
> Markdown is a double purpose format. Some people view it formatted:
> 
> https://github.com/AllureOfTheStars/Allure/blob/b0c0ffad460100aadf4d5ba3271ec4f9b1c1f68f/GameDefinition/PLAYING.md
> 
> other view it raw, e.g, in the console with a `cat` or `less` command
> 
> https://raw.githubusercontent.com/AllureOfTheStars/Allure/b0c0ffad460100aadf4d5ba3271ec4f9b1c1f68f/GameDefinition/PLAYING.md
> 
> Additionally, the latter form is used in the game itself. The help screen ends with this file included (as captured at compilation time), in raw form, and the start menu displays the initial portion, slightly formatted.
> 
> I'm afraid, to keep the 80 columns, one needs to use an old fashioned text editor with monospace font and a ruler or window size set to 80 columns. Then commit the changes using a commandline git. Alternatively, perhaps Visual Studio can do that, but I'm not familiar with it.

So, do I need to resize my inserts?
I can do that, I just thought it will be done automatically. But I can think how to rearrange it. It is an interesting coding task :)
Do I understand correct that I need to make the text as 80 symbols per raw and with spaces before the first letter of each raw, right?

Mikolaj commented:
> So, do I need to resize my inserts?
> I can do that, I just thought it will be done automatically. But I can think how to rearrange it. It is an interesting coding task :)

I'm afraid that's necessary and I'm afraid it can't be done automatically (not perfectly at least and not without ML), because you don't want to split phrases that go together, such as "to <verb>" or "<of noun>".

> Do I understand correct that I need to make the text as 80 symbols per raw and with spaces before the first letter of each raw, right?

The special spaces are only needed for the first paragraphs, the ones that go into the starting menu. The rest just needs the rows to be below 80 character long and, say, above 55.

dmytroshytikov commented:
> I'm afraid that's necessary and I'm afraid it can't be done automatically (not perfectly at least and not without ML), because you don't want to split phrases that go together, such as "to " or "".
> 
I'll see how to deal with this issue.

> 
> The special spaces are only needed for the first paragraphs, the ones that go into the starting menu. The rest just needs the rows to be below 80 character long and, say, above 55.
>
So, there is no need to have space in each raw, right? Just below 80 characters per raw, right?

dmytroshytikov commented:
> > So, there is no need to have space in each raw, right? Just below 80 characters per raw, right?
> 
> Yes. But if in doubt, please look at the surrounding text. There are no leading spaces. Why do you expect them to be there? Is this the misleading github display? (BTW, it's "row".)

Pardon me. The first part of the file was leading spaces.
Ok, I'll rearrange.

dmytroshytikov committed:
Update PLAYING.md

Does this work?

dmytroshytikov reviewed (commented):
Does this output work?

Mikolaj commented:
Let's see what CI says. ;)

dmytroshytikov commented:
Interesting, why it takes so much time now.

Mikolaj commented:
It probably doesn't fail at once, so probably it's fine now.

dmytroshytikov commented:
What do these command lines do? Do they process this text file in different ways?

Mikolaj commented:
Which lines, precisely?

dmytroshytikov commented:
These checks

Mikolaj commented:
To make sure we talk about the same thing: could you click on such a check you have in mind and copy here the link you followed and a fragment of the text you mean?

dmytroshytikov commented:
![image](https://user-images.githubusercontent.com/107208313/217198373-9518d989-12c0-4ff1-a175-7a0cb1ddd82b.png)
I know it is a strange question :)

Mikolaj commented:
The question is fine. It's just imprecise. I would never guess the "command lines" you mention are these CI job descriptions. Each line represents a single CI job that runs in the cloud. Did you try clicking on the link for any of the jobs? I think it would answer your question and it would not merge the PR, no worry. :) It says "Details", not "Merge".

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -431,10 +431,22 @@ at least 5% of damage always gets through, even if the bonus is nominally
-stats can be viewed in the skill menu, accessible via the `#` command,
-which summarizes all the stats conferred by organs and conditions listed
-in the organ menu, invoked by `@`.
+The current pointsman's melee bonus (~d~ + % where '~' stands for a number),
```
Later on we write "Weapon damage and other item properties are displayed using the dice notation `xdy`, which denotes `x` rolls of `y`-sided dice." Shall we standardize the notation? Either `~d~` in both places or `xyd` in both places? Or is this fine?

BTW, you can use "View file" from the right hand ... menu to see how the file is rendered in markdown. You'd notice that ~ without back quotes causes a text to be stroked through.

dmytroshytikov replied:
I forgot the notation. I will change. BTW:
1. is it reasonable to write xdy + z%? Most of weapons is xd1. That means there are no dice.
2. z% - does it the bonus to this weapon? For example, ceramic splinter cells are 1d1 - 80%. Does it mean I need 5 splinters to inflict one point of damage? And does a spade 8d1 -- 10% inflict 7.2 points of damage?

Mikolaj replied:
> is it reasonable to write xdy + z%?

Yes, I think so. Currently, `z` only appears in damage dice and then it's just percent of the rolled value added on top of it, as described in "The dice of the first recharged weapon, the one that is going to be used now, is adorned with percentage damage bonus collected from the whole equipment of the pointman". This is not well enough explained anywhere, so it makes sense to elaborate on that.

> Most of weapons is xd1. That means there are no dice.

Correct. Perhaps even all weapons. However, the engine supports weapons with non-trivial die, and also this resembles the common D&D and RPG notation, but underscores that there is no randomness, unlike in most other games.

> z% - does it mean the bonus to this weapon?

Yes, It affects the item it's written about. When displayed in item menu, it's only the percentage contributed by the weapon (or whatever the item is). But on HUD, it's the sum total that affects the weapon, summed from all equipment.

> For example, ceramic splinter cells are 1d1 - 80%. Does it mean I need to fling 5 cells to inflict one point of damage?

Yes. Thrown damage doesn't get bonuses from other equipment, so it will always be 80% for these splinters.

> And does a spade 8d1 -- 10% inflict 7.2 points of damage?

That's melee, but so if you wear nothing else, then 7.2, but if not, it can be different. The HUD displays that total. This is described somewhere, but without much detail, so worth discussion from a different angle and with different words.

Mikolaj replied:
I mean, the sums of bonus are on HUD, but also in the # menu, as you correctly mention in the text.

dmytroshytikov replied:
Another question:
A short club has 2d1 damage. But when the character equips it, it gives 2d1**+2** without percents. Does it mean that its damage was added to the character's natural one (I guess inflicted by the fists)?

Mikolaj replied:
No, it's about different kinds of damage. That must be described in the manual. Could you have a look? If it's not easy to find, we have to make it so.

Mikolaj replied:
BTW, that's also a key to an effective tank build, but the player is supposed to discover that by himself (as opposed to discovering UI meaning, which should be 100% clear and explicit from the docs).

dmytroshytikov replied:
Hm. So far, I have not found anything similar in the "Battling monsters" section. I'll look at it again later.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -431,10 +431,22 @@ at least 5% of damage always gets through, even if the bonus is nominally
+How to react to it? Is it dangerous? What are its properties? Check its
+lore menu and the organs it has! It may help you to generate the appropriate
+strategy. Because for example, the enemy that can call for help may be target
+#1. 
```
I'd rephrase, because markdown tries to interpret `#1` as a link and also because "target" and `#1` landed on different lines, which looks confusing.

dmytroshytikov replied:
sure.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
```
Given the new chapter, shall we remove "Q: How do I craft?"? Is the content of the answer contained in the new chapter?

dmytroshytikov replied:
I think it would be reasonable. Crafting is an important part of the game. Thus, it deserves to be in the main text. But it is just a suggestion. What do you think?

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+-------------------------------
+
+All items belonging to your team are stored either in the character's local
+inventory (called the character's outfit) or in the shared inventory that
```
I'd stick names with mutually disjoint sets of words, that is, any abbreviations of these full forms: "personal equipment outfit", "shared inventory stash". Notice that this matches the keys used for management. If you say "local inventory" the character may try to use key I to view it.

dmytroshytikov replied:
True. I'll check it again.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+is accessible by every team member (called the team stash). Items that are
+in the character's outfit can be accessed by or are beneficial for this
+particular character only. Stashed items can be accessed by the team members
+(unless the character is injured or panicking). But no one has a direct
```
To be more precise "is not calm enough" or "is stressed out", "e.g., due to being injured or panicking". This is clearly visible in the Calm field in the map UI (and the `#` menu). Also, this only affects the equipment, e.g., putting items from stash for outfit. You can still move item from stash to floor even totally panicked.

dmytroshytikov replied:
I'll update. To my memory, I couldn't work with the stashed items when the calmness level was very low. I'll check (and rephrase the text).

dmytroshytikov replied:
Seems to be so. Like the character with the calmness level of 0 cannot take anything from the stash, but can drop it from there.

Mikolaj replied:
Yes, I think such character can also fling and trigger items from stash. Not from equipment, though.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+the map and may be looted by your enemies. So, it is wise to keep it safe
+or change its location depending on the current situation. It would be
+a pity to lose your belongings, valuables, and powerful projectiles by not
+taking care of them. Enemy factions may have their stashes on the map,
+and you may loot their stashes too. Enemy units may have items in their
```
I wonder if we are preventing the player from discovering the job of looting enemy stash on his own. After all, it's clearly marked in pink and a message appears when it's found, so perhaps let's not spell it out?

dmytroshytikov replied:
True... I'll remove this spoiler) But I think it's reasonable to say about the possibility of the stash to be looted.

Mikolaj replied:
I think we can compromise: say that the player's stash can be looted, but don't mention whether enemy stash can or cannot.

dmytroshytikov replied:
OK! Sounds good.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+outfits, and sometimes they may have some items stashed on the ground.
+To relocate the stash, first, you need to remove ALL of the items from it
+anywhere you want to make it empty. Then, any stashed item (or a pile
+of items) will automatically become the new stash. The process is a bit
+long, so you should plan stash relocation in advance.
```
I don't remember, but doesn't it take 2 turns? That's risky if standing next to an enemy, but not "long".

dmytroshytikov replied:
1. Remove all from the stash.
2. Stash all.
That's almost that place from where I contacted you on the game website :)

Mikolaj replied:
So is it just 2 turns? You can check the turn counter at the top of history display before and after.

dmytroshytikov replied:
Well, the long crawl (with a HUGE stash with over 3 pages of items) is over, and relocation of a small stash takes 2 turns.

Mikolaj replied:
Thank you for the confirmation. Otherwise, it'd probably require a change.

So it's rather "a couple of turns" than "several turns", right? Pardon my pedantry: it's an occupational disease.

dmytroshytikov replied:
No problem. 2 turns so far.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+of items) will automatically become the new stash. The process is a bit
+long, so you should plan stash relocation in advance.
+You should keep your eye on the item management stat in the character menu
+'#'. If this stat is too low (which may be caused by injury or panic), you
```
Again, only caused by Calm stat being too low and it only affects the outfit equipment.

Mikolaj reviewed (changes requested):
Well done. Such technical texts require a lot of work to get all the details right and also to condense and also to deduplicate and harmonize with the rest of the document, so this is going to take a lot of effort still.

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+To manage items on the ground (trigger or fling it directly not putting
+it into any inventory to save time), press 'G'.
+
+To pick up an item from the ground to the pointsman's outfit, press 'g'.
```
maybe... I'll rephrase

Mikolaj replied:
That would be surprising. However 'i' would stash instead of equipping.

dmytroshytikov commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+management menu. You may want to choose all items from the particular location
+by pressing '*'.
+
+To drop an item on the ground from the inventory (either stashed or from
+the personal outfit), press 'r'.
+To trigger the item's properties (inject an ampoule, activate a chip, eat
+a piece of food), press 't'.
+To throw (fling) the item somewhere, press 'f'.
+To put the item into the particular outfit (personal or shared), press 'o'
+(personal outfit) or 'i' (shared inventory).
```
Well, I thought it would be reasonable. But yes, it's quite a lot, and it duplicates the in-game help. For me, IMHO, it would be good.
But if it is not, I will remove. It just took for me a while to get used to the system. I'm more used to how it is arranged in Nethack or some other games like that. So, I wrote it in the way, as if I was writing to me from the past :)

But this fragment is redundant indeed. So,shall we remove it?

Mikolaj replied:
I'm fine removing. However, I'd like to understand why the help screen is was not enough in your case. The commands you list here are beyond the minimal command set listed first in the help screen. Perhaps it's too small a set? Are you using 'o' and 'i' often? Do you think they are important for a newcomer? How did you come up with this particular list and not any wider subset?

dmytroshytikov replied:
Well, there are many commands about dealing with the inventory, and playing with the personal/shared inventories is an important part of the game. It took a while to learn these things, and still, I am not used to some of the commands (and the 'TAB' command may crash the game sometimes).
I routinely use the 'i' and 'I' commands to navigate items between the outfits. I rarely use the 'TAB' command. Maybe, I'll start using it more often to look for that bug.

dmytroshytikov replied:
I have deleted this fragment. It is available in the in-game help. Thus, indeed, no need to duplicate.

dmytroshytikov commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+etc. You may also change pointsmen that are adjusting inventory by pressing
+the 'TAB' command.
+
+Note how lowercase item commands (stash item, equip item) place items into
+a particular item store, while uppercase item commands (manage Inventory,
+manage Outfit) open the management menu for a store. Once a store menu is
+opened, you can switch stores with `<` and `>`, so the multiple commands
+only determine the starting item store. Each store is accessible from
+the dashboard as well.
```
Just copied from the in-game help to make it complete.  Can remove, as it duplicates the in-game help.

Mikolaj replied:
I think it's fine to duplicate in-game help, if it serves a purpose, but not to duplicate a previous paragraph. My worry is that most people drift off after 3 sentences of a paragraph, three paragraphs of a chapter, three chapters of a manual. We have a very limited budget of player focus, so we have to be selective. Repetitions rarely are a good use of such limited budget, unless you hammer something crucial, like "dying is fine" or "think about each turn" or "you can move more than one character" or "you don't have to move more than one".

dmytroshytikov replied:
True story. Solved.

dmytroshytikov commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+only determine the starting item store. Each store is accessible from
+the dashboard as well.
+
+Quite often, you will need to modify your environment somehow. There are
+numerous ways how to do that, so try and experiment!
+You may craft items, change the terrain, cook food, upgrade or change items,
+and many other things!
+For example, to craft items, you need some basic components and a workshop
+station or any other terrain that may be used to modify items (a pool
+of water, a source of fire, etc). Stand next to the workshop station, drop
+the components, stand on them, press 'M', and point to the direction
+of the item-modifying tile. The components will be automatically consumed,
+and the resulting item will appear. For example, you may want to craft
+a club from a wooden stick and several steel scraps. Approach a workshop,
+drop the components, press 'M', and point to the workshop.
+Some crafting recipes may be found by pressing the '~' command while looking
+at the terrain you want to use.
```
I will split. I thought it would be good to explain this part.

dmytroshytikov replied:
Done

dmytroshytikov commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+will be unable to work with items either in the character's outfit or in
+the shared stash.
+
+To open the pointsman's outfit, press 'O' (if you want to see the pointsman's
+personal inventory outfit and manipulate items stored there) or 'i' (if
+you want to stash something in the shared inventory from the pointsman's
+outfit). NA outfit).
+To open the shared stash, press 'I'.
```
oh, a typo detected

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov reviewed (commented):
OK, how about this?

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov reviewed (commented):
Just one more small change

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +538,104 @@ influence the speed of HP attrition and if it outweighs the benefits
+the dashboard as well.
+
+Quite often, you will need to modify your environment somehow. There are
+numerous ways how to do that, so try and experiment!
+You may craft items, change the terrain, cook food, upgrade or change items,
+and many other things!
```
Nitpicks: "ways to do", "and do many".

dmytroshytikov replied:
Done.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +538,104 @@ influence the speed of HP attrition and if it outweighs the benefits
+the components, stand on them, press 'M', and point to the direction
+of the item-modifying tile. The components will be automatically consumed,
+and the resulting item will appear. 
+For example, you may want to craft a club from a wooden stick and several 
```
This is now well split. The repeated "For example" is distracting, though (repetition is a powerful literary device).

dmytroshytikov replied:
Well, a fresh eye can always see things better.

dmytroshytikov committed:
Update PLAYING.md

Then, better to get rid of it

dmytroshytikov reviewed (commented):
Then, better to get rid of this paragraph. It is too similar indeed.

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov reviewed (commented):
Another change

dmytroshytikov committed:
Clean ` vs '

I tried to clean apostrophes.

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov reviewed (commented):
I have changed the wrong apostrophes ( ' ) to the correct ones ( ` ). So, the formatting must become more consistent.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+team member. 
+Items that are in the personal equipment outfit can be accessed by or 
+are beneficial for this particular character only. Stashed items can be 
+accessed by the team members (unless the character is not calm enough 
```
I'd say "by all team members (except those that are not calm enough..."

dmytroshytikov replied:
Solved

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+Items that are in the personal equipment outfit can be accessed by or 
+are beneficial for this particular character only. Stashed items can be 
+accessed by the team members (unless the character is not calm enough 
+or injured). But no one has a direct advantage from having the items 
```
How about "direct stats advantage" or "direct benefit to skill level"?

dmytroshytikov replied:
What is it?

Mikolaj replied:
What is what? Skill?

dmytroshytikov replied:
Yes, what do you mean by these terms? 
"direct stats advantage", "direct benefit to skill level".

Mikolaj replied:
I mean that some items, for as long as worn, make skills (as seen in the # menu) of the character higher.

Did you mean something else by your "direct advantage"? I suppose also the ability to use an item in melee, if applicable? Anything else?

dmytroshytikov replied:
Oh, I see. 
Yes, exactly this. That items stored in the shared inventory do not give bonuses to the team. The statement is a bit vague, probably, but very simple.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+or injured). But no one has a direct advantage from having the items 
+in the shared inventory stash.
+The stashed items are exactly stashed, they are not simply laying on
+the ground (although the stash has a particular location on the map). You
+cannot operate the items from the stash, as if these items are just on
+the ground directly. To collect an item from the stash or drop it on the ground,
```
I find the sentence "You cannot operate the items from the stash, as if these items are just on the ground directly." spurious and confusing, though I know what you mean. Perhaps remove it and instead extend the one: "you should use commands that refer to the party stash, not items on the ground."

dmytroshytikov replied:
Changed. Hopefully, it will look better

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+The shared inventory stash is of unlimited size. It has a physical location on
+the map and may be looted. So, it is wise to keep it safe or change 
+its location depending on the current situation. It would be a pity to lose 
+valuables or powerful projectiles by not taking care of the shared stash. 
```
not to repeat "stash" for the nth time, might we say "of the common inventory", perhaps?

dmytroshytikov replied:
I have replaced the word in several places

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+its location depending on the current situation. It would be a pity to lose 
+valuables or powerful projectiles by not taking care of the shared stash. 
+
+To relocate the shared inventory stash, first, you need to remove ALL 
+of the items from it anywhere you want to make it empty. Then, any stashed
+item (or a pile of items) will automatically become the new stash. The process 
+takes several turns, so you should plan shared inventory stash relocation
+in advance.
```
I think this could still use some careful rewording. And I still think "a could" would be more precise than "several".

dmytroshytikov replied:
done

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+item (or a pile of items) will automatically become the new stash. The process 
+takes several turns, so you should plan shared inventory stash relocation
+in advance.
+You should keep your eye on the item management stat in the character menu
+`#`. If this stat is too low (which may be caused by an injury or low 
+calmness level), it will be difficult for the character to work with items in 
```
This is not a bad idea, perhaps low calmness should affect item management and not directly equipment availability. However, the it's set up right now is that some rare effects just temporarily lower the item management stat (you can see the effects and how long they are going to last in the organs menu). Also, some characters (e.g., animals and some aliens or robots) innately have too low item management stat to manage items.

Calmness and HP injury have nothing to do with this (HP loss indirectly affects equipment management only in that it lowers Calm).

dmytroshytikov replied:
Well, injury is not about the HP loss but also about being deafened or temporarily stunned (for example, by the explosion). I wanted to mention that the item managing stat may change, but not giving spoilers.

Mikolaj replied:
Oh, I see now. I thought "injury" is only HP loss, but you are right in dictionary sense, though not every temporary handicap needs to be an injury in real life (e.g., flashbang may not cause an injury, despite causing a loss of combat effectiveness). Is there a good near-synonym of injury that aligns better with short loss of a capability instead of a longer lasting bodily harm?

dmytroshytikov replied:
Perhaps, "Trauma" may work?
IMHO, "Injury" is neutral enough. I have checked synonyms briefly, but other words are more about the direct physical damage that can attribute to the HP loss.

Mikolaj replied:
Well, there is a perfectly accurate term, "debuff", but it's gaming jargon and so breaks immersion. "Affliction" may be? "Impairment"? "Impediment"? Is any of these used in the manual, in-game help or the source code in general? "Injury" is not bad, but if we can be more precise, let's try.

dmytroshytikov replied:
Probably, "debuff" is an accurate term, but I am not familiar with it. "Injury" or "trauma" IMHO are somewhat more atmospheric :)
What about "impairment"?

Mikolaj replied:
Here's what I've found used currently:

```
~/r/LambdaHack$ git grep -i3 affliction
GameDefinition/PLAYING.md-- Q: Why are there two 'weakened' conditions in the organ menu of my hero?
GameDefinition/PLAYING.md-
GameDefinition/PLAYING.md-A: Each team has a different recipe for their weakness brew.
GameDefinition/PLAYING.md:Consequently, multiple affliction by the concoction from a single team
GameDefinition/PLAYING.md:prolongs a single malady, but affliction by concoctions from many teams
GameDefinition/PLAYING.md-causes concurrent ailments, with compounded effects, but independent
GameDefinition/PLAYING.md-and short durations. The benefit of the mechanism is that
GameDefinition/PLAYING.md-it's possible to tell the perpetrator team of any ailment.
~/r/LambdaHack$ git grep -i3 Impairment
~/r/LambdaHack$ git grep -i3 Impediment
~/r/LambdaHack$ git grep -i3 Injury
~/r/LambdaHack$ git grep -i3 debuff
~/r/LambdaHack$ git grep -i3 trauma
GameDefinition/Content/ItemKind.hs--- Not flashbang, because powerful bang without fragmentation is harder
GameDefinition/Content/ItemKind.hs--- to manufacture (requires an oxidizer and steel canister with holes).
GameDefinition/Content/ItemKind.hs--- The bang would also paralyze and/or lower the movement skill
GameDefinition/Content/ItemKind.hs:-- (out of balance due to ear trauma).
GameDefinition/Content/ItemKind.hs-flashBomb = fragmentationBomb
GameDefinition/Content/ItemKind.hs-  { iname    = "magnesium ribbon"  -- filled with magnesium flash powder
GameDefinition/Content/ItemKind.hs-  , iflavour = zipPlain [BrYellow]  -- avoid @BrWhite@; looks wrong in dark
```

If "affliction", "ailment" and "malady" are not to your liking, "impairment" certainly is fine. I like "trauma" a lot, too.

Mikolaj replied:
For completeness, in Allure's content I've found another occurrence: 

```
GameDefinition/Content/ItemKind.hs:  , idesc    = "A vial of bright, frothing concoction. 
The best medicine that nature has to offer for wounds, ailments and mood swings."
```

dmytroshytikov replied:
I suggest "trauma". IMHO, it suggests that a negative event can happen during the fight. It is ambiguous enough to not give spoilers directly. And it is atmospheric.
But I can change "injury" to "malady", "ailment", "impairment", or "ailment".

Mikolaj replied:
Trauma it is.

dmytroshytikov replied:
OK!

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+the character's personal outfit or in the shared inventory stash.
+
+To open the pointsman's outfit, press `O` (if you want to see the pointsman's
+personal inventory outfit and manipulate items stored there) or `i` (if
```
I'd recommend to reserve "inventory" to taking about stash and use "equipment outfit" for the other kinds. Mnemonics.

dmytroshytikov replied:
I've changed in some places. But I think it is better to use the term 'shared inventory stash' more or less consistently.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+outfit).
+To open the shared inventory stash, press `I`.
+To open the whole team's belongings (personal and stashed), press `T`.
+To manage items on the ground (trigger or fling it directly not putting
+it into any inventory to save time), press `G`.
```
A useful remark.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+management menu. You may want to choose all items from the particular 
+location by pressing `*`.
+
+You may want to travel around the characters' inventories without leaving
```
Perhaps "cycle among" instead of "travel around". I'm not sure. But "travel" certainly is confusing.

dmytroshytikov replied:
changed

dmytroshytikov committed:
Update PLAYING.md

I have corrected the text a bit more:
`Pointsman` to `pointman`
`travel` to `cycle`
`stash` to `inventory`, but not everywhere. After all, it may be also confusing
Lines 548-550

dmytroshytikov reviewed (commented):
I have adjusted the style of the text according to the recommendations a little.

dmytroshytikov commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
+You may want to travel around the characters' inventories without leaving
+the inventory menu. To do that, you may press `<` or `>` to check different
+inventories: character's outfit, stashed inventory, objects on the ground,
+etc. You may also change pointsmen that are adjusting inventory by pressing
```
oops

Mikolaj commented:
> I have adjusted the style of the text according to the recommendations a little.

Thank you so much. Will review next week.

dmytroshytikov committed:
Update PLAYING.md

The limit of symbols per line was exceeded.

dmytroshytikov reviewed (commented):
Integration did not work due to the symbol per line limit. Now, it must work.

dmytroshytikov commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+the ground (although the stash has a physical location on the map). You
+cannot operate the items from the stash, as if these items are just on
+the ground directly. To collect an item from the stash or add it there,
+you should enter the stash management menu by pressing the 'I'.
```
Solved

dmytroshytikov commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,89 @@ influence the speed of HP attrition and if it outweighs the benefits
+the inventory menu. To do that, you may press `<` or `>` to check different 
+inventories: character's outfit, stashed inventory, objects on the ground, etc. 
+You may also change pointmen that are adjusting inventory by pressing 
+the `TAB` command.
```
Considering the fact that this command caused the game to crash, is it reasonable to keep it functional? Maybe, it is better to disable it? Personally me, I have never used it.

Mikolaj replied:
I'm afraid, to disable it, we'd need to be able to compile the game and release a new version. But it's already fixed (I mean the crash, the occasional lack of any effect is not, if that's a bug) in the new version. However, feel free to remove the sentence describing it. It's quite obvious what it does, so no point keeping it, especially that it's broken.

dmytroshytikov replied:
Well, I do not mind against this fragment. Shall I keep it?

Mikolaj replied:
Delete it. It's, in some form, in in-game help anyway.

dmytroshytikov replied:
done.

dmytroshytikov reviewed (commented):
Just one more comment about the `TAB` command.

dmytroshytikov committed:
Update PLAYING.md

I removed the `TAB` sentence.

dmytroshytikov reviewed (commented):
I removed the `TAB` sentence (line 603-604). The rest seems to be fine.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -431,10 +431,22 @@ at least 5% of damage always gets through, even if the bonus is nominally
+to lose control over your teammate just due to not finding out that he/she
+was terrified by some monstrosity that threatened and impressed him/her
+some time ago.
+The same information about enemies may be found by the `~` command while
```
I'd say "via the ... command" or "using" even "with".

dmytroshytikov replied:
Changed

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -431,10 +431,22 @@ at least 5% of damage always gets through, even if the bonus is nominally
+The same information about enemies may be found by the `~` command while
+the enemy unit is in the crosshair. For example, you see an unknown creature.
+How to react to it? Is it dangerous? Check its lore menu and the organs 
+it has! It may help you to generate the appropriate strategy. For example, 
```
I guess "come up" or "create" or "invent" is more natural than "generate"

dmytroshytikov replied:
changed

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -431,10 +431,22 @@ at least 5% of damage always gets through, even if the bonus is nominally
+the enemy unit is in the crosshair. For example, you see an unknown creature.
+How to react to it? Is it dangerous? Check its lore menu and the organs 
+it has! It may help you to generate the appropriate strategy. For example, 
+the enemy that can call for help may be the main target for you. 
```
This is fine, but I'd write "may become". Feel free to ignore.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+Items that are in the personal equipment outfit can be accessed by or 
+are beneficial for this particular character only. Stashed items can be 
+accessed by all the team members except those that are not calm enough 
+or injured somehow. But no one has a direct advantage from having the items 
```
Did we agree on "or suffer a debilitating trauma" or something like that?

dmytroshytikov replied:
Yeah... Pardon me.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+item), it is reasonable to put it into a particular character's outfit.
+However, you should remember that the outfit is limited to any 10 (ten)
+items only. On the contrary, if the item is not required directly (ranged
+weapon, medkits, ampoules with drugs, food, or else), it may be stored in
```
I'd say "it may well be stored" to hint that I'm not suggesting that other kinds of items "may not" be stored. The art of technical writing (which I'm bad at) is to predict and eliminate confusion before it arises.

dmytroshytikov replied:
done

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+its location depending on the current situation. It would be a pity to lose 
+valuables or powerful projectiles by not taking care of the shared inventory. 
+
+To relocate the shared inventory stash, first, you need to remove ALL 
+of the items from it anywhere you want to make it empty. Then, any stashed
```
Is some of this an old text that didn't get removed?

dmytroshytikov replied:
No, just a description of how to relocate the stash

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+item (or a pile of items) will automatically become the new stash. The process 
+takes a couple of turns, so you should plan shared inventory stash relocation
+in advance.
+You should keep your eye on the item management stat in the character menu
```
Add new paragraph?

dmytroshytikov replied:
I think no need. Or better to add?

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+takes a couple of turns, so you should plan shared inventory stash relocation
+in advance.
+You should keep your eye on the item management stat in the character menu
+`#`. If this stat is too low (which may be caused by an injury or low 
```
trauma?

dmytroshytikov replied:
Yeah...

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+You should keep your eye on the item management stat in the character menu
+`#`. If this stat is too low (which may be caused by an injury or low 
+calmness level), it will be difficult for the character to work with items in 
+the character's personal outfit or in the shared inventory stash.
```
to be precise, the character can still melee with the items, so perhaps: "add or remove items to the character's personal outfit". This also covers moving items from stash to outfit, so the "or in the shared inventory stash" can be skipped.

dmytroshytikov replied:
Done

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+calmness level), it will be difficult for the character to work with items in 
+the character's personal outfit or in the shared inventory stash.
+
+To open the pointman's outfit, press `O` (if you want to see the pointman's
+personal inventory outfit and manipulate items stored there) or `i` (if
+you want to stash something in the shared inventory from the pointman's
+outfit).
```
I don't understand. "To open the pointman's outfit, press `O` ... or `i`"? Are you sure `i` opens that menu?

dmytroshytikov replied:
`O` open the outfit, `i` suggests to stash the item.

Mikolaj replied:
Right, `i` asks what to stash either from the ground, or from the outfit, depending on whether you stand over an item or not. I general, as said in the help screens, `O` lists outfit, `o` asks what to put into outfit, `I` lists inventory, `i` asks what to put into inventory. The container *from which* to take items is not determined by the key, but by other factors.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+outfit).
+To open the shared inventory stash, press `I`.
+To open the whole team's belongings (personal and stashed), press `T`.
+To manage items on the ground (trigger or fling it directly not putting
```
Missing comma after "directly"

dmytroshytikov replied:
done

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+it into any inventory to save time), press `G`.
+
+To pick up an item from the ground to the pointman's outfit, press `g`
+or `,`. Items that have some direct use (melee weapon or armor) 
```
"An item that has any", to match "it has" later on.

dmytroshytikov replied:
Done

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+
+When you are managing items in the inventory or on the ground, you should
+choose the item with the LMB or arrows and `RET` buttons to get the item
+management menu. You may want to choose all items from the particular 
```
"You can choose"

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+location by pressing `*`.
+
+You may want to cycle among the characters' inventory outfits without leaving 
+the inventory menu. To do that, you may press `<` or `>` to check different 
```
"may" is spurious here IMHO. Also, I'd merge this paragraph with the previous one.

dmytroshytikov replied:
Done

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+You may craft items, change the terrain, cook food, upgrade or change items,
+and do many other things!
+
+To craft or modify items, you need some basic components and a workshop
+station or any other terrain that may be used to modify items (a pool
```
I like "workshop station". A nice invention.

dmytroshytikov replied:
I wanted to use a more general term. Because we can use water, fire, or oil spills to modify some items.

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+
+To craft or modify items, you need some basic components and a workshop
+station or any other terrain that may be used to modify items (a pool
+of water, a source of fire, etc). Stand next to the workshop station, drop
+the components, stand on them, press `M`, and point to the direction
```
The "stand on them" is a bit misleading. The character is already standing on them, right?

"point in the direction"

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+For example, you may want to craft a club from a wooden stick and several 
+steel scraps. Approach a workshop, drop the components, press `M`, and 
+point to the workshop.
+Some crafting recipes may be found by pressing the `~` command while looking
+at the terrain you want to use.
```
I'd drop "Some", because all can be found in this way.

Perhaps it's worth it to preserve a couple of details from this removed text? E.g, here mention that "the lore menu, terrain submenu" shows the recipes for encountered terrain.

Stand adjacent to the workshop terrain (blue colon on the map) and experiment. Note that walkable positions can't be activated by bumping and, anyway, crafting requires the Modify command (key M) even for terrain that could be bumped into. Other crafting spots (e.g., any water and fire) work similarly, but the recipes differ. Read the recipes in the lore menu, terrain submenu, or by pressing ~ when crosshair points at terrain that enables crafting.

Mikolaj replied:
I meant the (repaired) spacesuit. I've just checked, it's there among the recipes of the bench of crafting (the workshop). I do think no are secret in the sense that the recipe is not visible to the user. However, some are hard to craft and until you craft the first specimen, the item does not show in the item lore, so its properties are secret.

Mikolaj replied:
Regarding line the whole crafting subsection, I just wondered aloud whether some of the old text would be useful, since not all is already contained in the new text.

Mikolaj reviewed (changes requested):
Really close now. Well condensed.

dmytroshytikov committed:
Update PLAYING.md

I have added most of the suggestions. However, I am unclear about what to change in lines 608-618.

dmytroshytikov reviewed (commented):
I have added the suggested changes except for lines 608-618. I am unclear of what to do there.

dmytroshytikov committed:
Update PLAYING.md

I think it would be more reasonable to keep the crafting section in the main help. I have cut a few lines that were definitely redundant.
But I don't mind if crafting moves to the Q&A section.

dmytroshytikov committed:
Update PLAYING.md

During the previous attempt, I failed to incorporate changes. Let me try once again.

dmytroshytikov reviewed (commented):
I have reshaped the crafting section slightly.

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov reviewed (commented):
reshaping again

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov reviewed (commented):
Reshape. Something blocks merging

dmytroshytikov committed:
Update PLAYING.md

dmytroshytikov reviewed (commented):
reshape

dmytroshytikov committed:
Update PLAYING.md

Reshaping lines 611 and 612

dmytroshytikov reviewed (commented):
Reshaping lines 611 and 612

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,80 @@ influence the speed of HP attrition and if it outweighs the benefits
+station or any other terrain that may be used to modify items (a pool
+of water, a source of fire, etc). Stand next to the item-modifying tile 
+(workshop station, source of fire or water), drop the components on
+the ground, press `M`, and point in the direction of that tile. The 
```
```suggestion
the ground, press `M`, and point in the direction of that tile. The
```

Mikolaj committed:
Update GameDefinition/PLAYING.md

Mikolaj commented on GameDefinition/PLAYING.md:
```diff
@@ -527,6 +539,80 @@ influence the speed of HP attrition and if it outweighs the benefits
+
+To craft or modify items, you need some basic components and a workshop
+station or any other terrain that may be used to modify items (a pool
+of water, a source of fire, etc). Stand next to the item-modifying tile 
```
```suggestion
of water, a source of fire, etc). Stand next to the item-modifying tile
```

dmytroshytikov replied:
Strange. I didn't put there anything unusual.

Mikolaj committed:
Update GameDefinition/PLAYING.md

dmytroshytikov committed:
Update PLAYING.md

Remove white spaces from the ends of all of the lines

dmytroshytikov reviewed (commented):
Remove whitespaces from the ends of all of the lines 542-613

Mikolaj reviewed (approved):
LGTM

Thank you!

Mikolaj commented:
Well done. Your new version of the manual is now the one shown by the main manual hyperlinks. Only the in-game manual is the old one until a new release is made (which may take a while).
